package.json

```diff
@@ -102,5 +102,8 @@
     "webpack-dev-server": "^1.10.0",
     "webpack-stats-plugin": "^0.1.1"
   },
-  "sideEffects": false
+  "sideEffects": false,
+  "dependencies": {
+    "circle2": "^1.1.1"
```

Comment:
I would really prefer not to take this dependency. Can you get it done without?

Reply:
I can give it a try! Do you have some general guidelines on whether or not to take a dependency, like a best practices guide? I didn't see anything about that in the `formidable-playbook`.

Reply:
Assuming we did want this dep, isn't it supposed to be in https://github.com/FormidableLabs/victory/blob/master/packages/victory-bar/package.json ?

Reply:
Yes, keeping it in the main package.json would add it to every package.

Reply:
Thanks for pointing that out! I'll uninstall it from the main package.json.